programs/programs/nc/nc_aca/calculator.py

```diff
@@ -0,0 +1,49 @@
+from programs.programs.calc import ProgramCalculator, Eligibility
+from programs.programs.helpers import medicaid_eligible
+import programs.programs.messages as messages
+from integrations.services.sheets import GoogleSheetsCache
+
+
+class ACACache(GoogleSheetsCache):
+    default = {}
+    sheet_id = "1tk8zfO_Ou96UvGrIwZoI3Pv8TvPZZipg7YfzGMT2o3c"
+    range_name = "'2023 report'!A2:B101"
```

Comment:
I would recommend changing the sheet name to "current". It makes it easier to make updates in the future, because you can just update the "current" sheet, so you don't have to make any code changes, or have a tab with the year 2023, but the data is for 2024.